### ISSUE TITLE
Sisters of Mercy buff.

### ISSUE THREAD

**Issue by Tat011** [captaincobbs/ESG-Mod#1496]
As Waky said this minor is by far the least useful so they need a buff, that being said we dont wanna buff Vaulters so its a bit tricky.
Arphar proposed giving them sterile food "so vaulters can usually use the systems they get their hands on regardless of composition" and i think that's probably the healthiest option (i assume +2? idk the numbers exactly).

**Comment by wakystuf** [comment]
Yeah, that's probably a good idea. That shouldn't provide the Vaulters with much more value than the pop that they'd otherwise have, so should be fine. +2 should be fine (doubled if boosted); could be any FIDSI.
I think their pop collection bonuses are also terrible, right? I forget what they are, but maybe something about ground defenses or healing rate (both of which are terrible)?

**Comment by Tat011** [comment]
Trait 1:
+50% Hero regen per turn. (Should probably be buffed to +100% assuming its not getting replaced)

Trait 2:
+10% Regen on Ships. (outside of fights) (Should probably also be buffed to +100%, even then it still seems pretty useless)

**Comment by wakystuf** [comment]
Yeah, wow those are terrible. If we made them +1000% I'd still ban them from breeding. Unless the ship regen works inside hostile territory, in which case we should just remove it due to risking breaking things

**Comment by Tat011** [comment]
uhh, im not sure, theres a natural regen always applied to ships and it probably modifies that? but idk

**Comment by Tat011** [comment]
oh these are the assimilation traits, not collection

**Comment by Tat011** [comment]
Collection are acording to the wiki:
+20% Health Regen (ship) Ratio on Systems with Sisters of Mercy

Increases Health of Troops when Defending in Ground Battle by +25%

**Comment by Tat011** [comment]
Thats weird
`<SimulationDescriptor` Name="PopulationCollectionBonusSistersOfMercy2" Type="PopulationCollectionBonus">
`<Modifier` TargetProperty="DockedFleetsHealthRegenerationRatioBonusFromCollection" Operation="Addition" Value="0.20" Path="../ClassEmpire,EmpireTypeMajor/ClassStarSystem/ClassPopulationStarSystemAffinitySistersOfMercy" />
`</SimulationDescriptor>`

`<SimulationDescriptor` Name="PopulationCollectionBonusSistersOfMercy3" Type="PopulationCollectionBonus">
`</SimulationDescriptor>`

**Comment by wakystuf** [comment]
For assimilation then those are fine. Those pop collection bonuses are even worse though; I'd forgotten how bad they were since I remember going through it carefully a long time ago and imposing a hard ban on ever growing those pop.

**Comment by Tat011** [comment]
We could just give them influence on T2 collection like Kalgeros and for T3 we give the Health to attacking troops too

**Comment by wakystuf** [comment]
Yeah those are both reasonable I think

**Comment by captaincobbs** [comment]
Can we give them approval as a T2? More accurate to their bonuses in Endless Legend, influence makes no sense for them. Their law is +25% Health on Troops, could make it an increase to the morale effect on ships or something weird, or reduced manpower damage on ships.

**Comment by captaincobbs** [comment]
Sorry for being a stickler about it, but you should look at their Endless Legend skill tree and see if anything looks particularly interesting, I'd like to keep themes the same even if we are changing the bonuses
Link to skill tree: https://endlesslegend.fandom.com/wiki/Valete_Sapiri_Kugua

**Comment by Tat011** [comment]
ok so aparently approval as a collection is a thing though oddly enough only guardians use it so ill change it

**Comment by Tat011** [comment]
I think this is complete now?
To recap the changes are:
-Giving the pop +2 food on sterile
-Changing its 2nd collection bonus from +50% ship health regeneration when in systems with this pop to +10 approval in systems with this pop.
-Changing the 3rd collection bonus from +35% HP to troops when defending and +10% ship HP to +25% HP to troops period both with the law setting on and off.
